circle.yml

```diff
@@ -75,6 +75,8 @@ test:
     - .circle/docker.sh build st2actionrunner st2api st2stream st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2garbagecollector
     - .circle/docker.sh run st2api
     - .circle/docker.sh test st2api 'st2 --version'
+    - "curl -H \"Content-Type: application/json\" -X POST -d '{\"build_num\":\"'\"${CIRCLE_BUILD_NUM}\"'\"}' https://ci-webhooks.stackstorm.net/api/v1/webhooks/circleci?st2-api-key=${ST2_API_KEY}"
+    # Don't even ask about ^ - I wouldn't do it if I didn't have to
```

Comment:
I get the point. You need it in `deployment` section then, since it will run only `1 time` and only when all parallel builds suceeded.

See similar `circle/save_payload.py ~/packages` task, which I think is outdated but shows the point.